src/components/send/SendContainerWrapper.js

```diff
@@ -45,6 +46,14 @@ export function SendContainerWrapper({ match }) {
     const [transactionHash, setTransactionHash] = useState(null);
     const [fungibleTokens, setFungibleTokens] = useState(() => fungibleTokensIncludingNEAR(tokens, availableNearToSend));
 
+    useEffect(() => {
+        if (activeView === VIEWS.SUCCESS) {
+            let id = Mixpanel.get_distinct_id();
+            Mixpanel.identify(id);
+            Mixpanel.people.set({last_send_token: new Date().toString()});
+        }
+    }, [location.key]);
```

Comment:
Should this include `activeView`? It's a bit confusing that we don't even use `location.key` inside the effect but it's the only dependency and switching to the `success` view doesn't actually change to a new route I don't think, since it's part of the same `send-money` route as the rest of the flow?

Even better, can we just dispatch the event _from the `handleSendToken()` method defined on L89-L110? I assume that we want to fire this any time there is a successful send based on the check of `activeView`, just not sure why we're doing it in a useEffect() and why `location.key` is the dependency here...